indexer/editable_map_object.cpp

```diff
@@ -52,6 +52,19 @@ size_t PushMwmLanguages(StringUtf8Multilang const & names, vector<int8_t> const
   
   return count;
 }
+
+string const kWebSiteProtocols[] = {"http://", "https://"};
```

Comment:
Use char const * const instead of strings - in any case you use c_str. Moreover, char const * const do not require string::ctor() call.